proposed/phpdoc-tags.md

```diff
@@ -1072,8 +1072,7 @@ specific type of exception.
 The @throws tag MAY be used to indicate that "Structural Elements" throw a
 specific type of error.
 
-The type provided with this tag MUST represent an object of the class Exception
-or any subclass thereof.
+The type provided with this tag MUST represent an object of the class Exception, or any subclass thereof, or an object that is a subtype of Throwable.
```

Comment:
Maybe we should just go with a blanket `Throwable`, and leave it at that?  I think Throwable might have come along some time after Mike originally wrote this section.